calibration/ADCPSL/CGINS-ADCPSL-24664__20220408.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24664,CC_scale_factor1,0.45,constant; CP02PMUO-00019
+24664,CC_scale_factor2,0.45,constant
+24664,CC_scale_factor3,0.45,constant
+24664,CC_scale_factor4,0.45,constant
+24664,CC_bin_size,800,in cm
+24664,CC_dist_first_bin,1695,in cm
+24664,CC_orientation,1,1 is upward looking; 0 is downward looking
```

Comment:
OK
I don't think it matters, but this one has an orientation of 1, while most others have 1.0